wagtail/admin/templates/wagtailadmin/home.html

```diff
@@ -6,17 +6,8 @@
     {{ block.super }}
     {{ media.css }}
 {% endblock %}
-
 {% block content %}
-    <header class="header merged header--home">
-        <div class="avatar"><img src="{% avatar_url user %}" alt="" /></div>
-
-        <div class="sm:w-ml-4">
-            <h1 class="header__title">{% block branding_welcome %}{% blocktrans trimmed %}Welcome to the {{ site_name }} Wagtail CMS{% endblocktrans %}{% endblock %}</h1>
-            <div class="user-name">{{ user|user_display_name }}</div>
-        </div>
-    </header>
-
+    {% include "wagtailadmin/shared/header.html" with title=site_name avatar=1 username=user|user_display_name merged=1 %}
```

Comment:
as noted below, it would be better to pass in the actual avatar URL to avoid the shared header template needing to have the overhead of using the template tag itself.

if `{% avatar_url user as avatar %}` does not work, you may need to pass it down from the template view.

```suggestion
    {% avatar_url user as avatar %}
    {% include "wagtailadmin/shared/header.html" with title=site_name avatar=avatar subtitle=user|user_display_name merged=1 %}
```

Also, as noted - try to keep the existing 'slots' where possible, this may look and work great with the subtitle being the username (but it may need some styling tweaks specific to this page)